data/policies/systems.cas

```diff
@@ -0,0 +1,7 @@
+system name {
```

Comment:
Is this actually being consumed anywhere?  I think we may need to add a test that this builds.

If you use valid_policy_test(), secilc will complain that all policies must contain an allow rule, so you would need to add a domain or resource into the module and an allow() call into that in order to make this a valid policy.

It would be good to verify the configs and the policy name (which the policy name isn't checkable via valid_policy_test(), so that will need special handling.